Makefile

```diff
@@ -58,27 +106,8 @@ edit:
 	$EDITOR src/lib.rs &
 	godot -e &
 
-release: build-release clean
-	godot --export "Android" ./native/bin/android/{{project-name}}.apk
-	godot --export "Linux/X11" ./native/bin/linux-x11/{{project-name}}.x86_64
-
-release-debug: build clean
-	godot --export-debug "Android" ./native/bin/android/{{project-name}}.debug.apk
-	godot --export-debug "Linux/X11" ./native/bin/linux-x11/{{project-name}}.debug.x86_64
-
-release-android: build-aarch64-linux-android-release build-armv7-linux-androideabi-release build-i686-linux-android build-x86_64-linux-android clean
-	godot --export "Android" ./native/bin/android/{{project-name}}.apk
-
-release-android-debug: build-aarch64-linux-android build-armv7-linux-androideabi build-i686-linux-android-release build-x86_64-linux-android-release clean
-	godot --export-debug "Android" ./native/bin/android/{{project-name}}.debug.apk
-
-release-linux-x11: build-x86_64-unknown-linux-gnu-release clean
-	godot --export "Linux/X11" ./native/bin/linux-x11/{{project-name}}.x86_64
-
-release-linux-x11-debug: build-x86_64-unknown-linux-gnu clean
-	godot --export-debug "Linux/X11" ./native/bin/linux-x11/{{project-name}}.debug.x86_64
-
-run: build-x86_64-unknown-linux-gnu
+run:
+	make build-x86_64-unknown-linux-gnu-debug
```

Comment:
Can we make this dependent on the user platform somehow? A lot of users are developing on Windows.

Reply:
That what I was thinking as well, currently `cargo-generate` does not support this, I filed an issue with cargo-generate [here](https://github.com/ashleygwilliams/cargo-generate/issues/251)

Reply:
my [PR](https://github.com/ashleygwilliams/cargo-generate/pull/252) has been merged, lest just wait for the release. I will create a separate issue for this.